common/components/Input.js

```diff
@@ -69,6 +71,13 @@ const Input = ({
         inputIsValid = false;
         setErrorMessage(`최대 값은 ${max}${unit} 입니다`);
       }
+
+      if (value.indexOf('.') > dotIndex) {
```

Comment:
includes 메소드를 써도 좋을 것 같습니다
```suggestion
      if (value.includes(".")) {
```